test/integration/ovs/ofctrl_test.go

```diff
@@ -240,7 +240,13 @@ func TestOFctrlFlow(t *testing.T) {
 		if err != nil {
 			t.Errorf("Failed to DeleteFlowsByCookie: %v", err)
 		}
-		flowList, _ = OfctlDumpTableFlowsWithoutName(ovsCtlClient, myTable.GetID())
+		wait.PollImmediate(time.Millisecond*100, time.Second, func() (done bool, err error) {
+			flowList, _ = OfctlDumpTableFlowsWithoutName(ovsCtlClient, myTable.GetID())
+			if len(flowList) > 0 {
+				return false, nil
+			}
+			return true, nil
+		})
 		if len(flowList) > 0 {
 			t.Errorf("Failed to delete flows by CookieID")
 		}
```

Comment:
```suggestion
		require.NoError(wait.PollImmediate(time.Millisecond*100, time.Second, func() (done bool, err error) {
			flowList, err = OfctlDumpTableFlowsWithoutName(ovsCtlClient, myTable.GetID())
			require.Nil(t, err)
			return len(flowList) == 0, nil
		}), "Failed to delete flows by CookieID")
```
To avoid returning empty flowList because of unexpected error, and leverage the result of `PollImmediate`.

Reply:
updated.